simulators/utils/predictive_utils.py

```diff
@@ -5,78 +5,7 @@
 import logging
 
 from inferential_models.batter_runs_models import BatterRunsModel
-
-
-def calculate_bowling_probabilities(all_innings_df) -> pd.DataFrame:
```

Comment:
Take away all the complex logic to calculate the next bowler, and resort to a simple random selection